app/serializers/api/v0/enterprise_serializer.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V0::EnterpriseSerializer < ActiveModel::Serializer
+  attributes :id, :url, :name, :email, :website, :category
+  attributes :description, :long_description
+
+  has_one :address
+
+  def url
+    enterprise_url(object)
```

Comment:
Groups are not enterprises. That's a different model. But yes, we need to distinguish between producers and shops here.